### ISSUE TITLE
Nice but difficult

### ISSUE THREAD

**Issue by notflip** [jaiwalker/setup-laravel5-package#1]
Hi,

Thanks for the guide, it is however weirdly written.. You have the example serviceprovider we're creating the jai/contact thing, And all of a sudden your adding the illuminate/html serviceprovider as an example.. It's really hard to follow

Thank you

**Comment by jaiwalker** [comment]
I am extremely sorry , Thats  an error  .Yes,you need to  have 'Jai\Contact\ContactServiceProvider', in app config providers instead of  'illuminate/html serviceprovider ', I will make this change.
Thanks for writing this.

If  you want a create a Laravel 5 package , use this https://github.com/jaiwalker/laravel5-package-creator-command

Once you get this - all you need to do is " create-package vendorName PackageName " where vendorName is eg: jai  package name is "contact" - try this .
